src/lib/editor.ts

```diff
@@ -1,20 +1,37 @@
-export type SupportedEditor =
-	| 'vscode'
-	| 'phpstorm'
-	| 'cursor'
-	| 'windsurf'
-	| 'nova'
-	| 'webstorm'
-	| 'sublime'
-	| 'atom';
+export type SupportedEditor = 'vscode' | 'phpstorm' | 'cursor' | 'windsurf' | 'webstorm';
+
+export type SupportedEditorConfig = {
+	label: string;
+	url: ( path: string ) => string;
+};
 
 export const supportedEditorNames: Record< SupportedEditor, string > = {
 	vscode: 'Visual Studio Code',
 	phpstorm: 'PhpStorm',
 	cursor: 'Cursor',
 	windsurf: 'Windsurf',
-	nova: 'Nova',
 	webstorm: 'WebStorm',
-	sublime: 'Sublime',
-	atom: 'Atom',
+};
+
+export const supportedEditorConfig: Record< SupportedEditor, SupportedEditorConfig > = {
+	vscode: {
+		label: 'VS Code',
```

Comment:
We don't want to translate the label. So it's fine to keep here as config.

Reply:
Can we make those translatable, but do not translate them by default? See https://github.com/Automattic/studio/pull/48 for context.

Also, should we keep the names in sync so we have VS Code in both places instead of Visual Studio Code in one place and VS Code in another?

Reply:
It seems to me that `supportedEditorConfig` has the same mappings as `supportedEditorNames` but it contains more details.

`supportedEditorNames` has only been used here previously:
https://github.com/Automattic/studio/blob/93adc785c3485fd0c6f5fca7c946be4ecddf2792/src/modules/user-settings/components/editor-picker.tsx#L36-L42

 I'd suggest we refactor the usages of `supportedEditorNames` to use `supportedEditorConfig` instead, and we can remove `supportedEditorNames` altogether.